src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachLayout.tsx

```diff
@@ -0,0 +1,256 @@
+  });
+
+  const listOfBreaches = exposedData.map((item: SubscriberBreach) => (
+    <div key={item.id} className={styles.breachItem}>
+      {/* TODO: Localize all date values MNTOR-2111 */}
+      {l10n.getFragment("high-risk-breach-name-and-date", {
+        elems: { breach_date: <span className={styles.date} /> },
+        vars: {
+          breach_name: item.name,
+          breach_date: dateFormatter.format(new Date(item.addedDate)),
+        },
+      })}
+    </div>
+  ));
+
+  const breachList = (
+    <div className={styles.breachItemsWrapper}>{listOfBreaches}</div>
+  );
+
+  let title, secondaryDescription, recommendationSteps, breachIllustration;
+
+  const CreditCardRecommendationSteps = (
+    <ol>
+      <li>{l10n.getString("high-risk-breach-credit-card-step-one")}</li>
+      <li>{l10n.getString("high-risk-breach-credit-card-step-two")}</li>
+      <li>{l10n.getString("high-risk-breach-credit-card-step-three")}</li>
+    </ol>
+  );
+
+  const BankAccountRecommendationSteps = (
+    <ol>
+      <li>{l10n.getString("high-risk-breach-bank-account-step-one")}</li>
+      <li>{l10n.getString("high-risk-breach-bank-account-step-two")}</li>
+      <li>{l10n.getString("high-risk-breach-bank-account-step-three")}</li>
+    </ol>
+  );
+
+  const SocialSecurityNumberRecommendationSteps = (
+    <ol>
+      <li>{l10n.getString("high-risk-breach-social-security-step-one")}</li>
+      <li>{l10n.getString("high-risk-breach-social-security-step-two")}</li>
+    </ol>
+  );
+
+  const pinRecommendationSteps = (
+    <ol>
+      <li>{l10n.getString("high-risk-breach-pin-step-one")}</li>
+      <li>{l10n.getString("high-risk-breach-pin-step-two")}</li>
+      <li>{l10n.getString("high-risk-breach-pin-step-three")}</li>
+    </ol>
+  );
+
+  // TODO: Expose email list & count here https://mozilla-hub.atlassian.net/browse/MNTOR-2112
+  const emailsMonitored = ["email1@gmail.com", "email2@gmail.com"]; // mocked
+  const emailsFormatter = new Intl.ListFormat("en", {
+    style: "long",
+    type: "conjunction",
+  });
+
+  switch (props.typeOfBreach) {
+    case "creditCard":
+      title = l10n.getString("high-risk-breach-credit-card-title");
+      secondaryDescription = (
+        <p>{l10n.getString("high-risk-breach-credit-card-description")}</p>
+      );
+      recommendationSteps = CreditCardRecommendationSteps;
+      breachIllustration = CreditCardIllustration;
+      break;
+    case "ssnBreaches":
+      title = l10n.getString("high-risk-breach-social-security-title");
+      secondaryDescription = (
+        <p>{l10n.getString("high-risk-breach-social-security-description")}</p>
+      );
+      recommendationSteps = SocialSecurityNumberRecommendationSteps;
+      breachIllustration = SocialSecurityNumberIllustration;
+      break;
+    case "bankAccount":
+      title = l10n.getString("high-risk-breach-bank-account-title");
+      secondaryDescription = (
+        <p>{l10n.getString("high-risk-breach-bank-account-description")}</p>
```

Comment:
The [design](https://www.figma.com/file/CaEKIhvSJqf6KNIMzSkt40/Concepts-for-Monitor-MVP-Redesign?type=design&node-id=546-40774&mode=design&t=eac5o4tyNTG821wQ-0) is showing this paragraph before the bullet list. Did we change the order?